project/Dependencies.scala

```diff
@@ -33,7 +33,7 @@ object Dependencies {
 
   val logstashLogback: ModuleID = "net.logstash.logback"      % "logstash-logback-encoder" % "6.6"
   val logbackClassic: ModuleID = "ch.qos.logback"             %  "logback-classic" % "1.2.10"
-  val ravenLogback: ModuleID =   "com.getsentry.raven"        %  "raven-logback"   % "7.8.6"
+  val ravenLogback: ModuleID =   "com.getsentry.raven"        %  "raven-logback"   % "8.0.3"
```

Comment:
https://docs.sentry.io/platforms/java/guides/logback/legacy/migration/

We can make the version upgrade as described on the PR, that's fine, but I'm wondering if we shouldn't also make the changes described here?  I'm worried that if 8.0.x is the final raven-* version, will scala steward be smart enough to know that the overall package name changed for future updates?  I suspect not.

Reply:
good catch! did you already know about the raven -> sentry name change or were your spidey senses tingling?